canonical-web-and-design/maas-ui
# fix(legacy) null check rrsets in domain

## Done
Add extra null checks for the existence of `rrsets` when asserting if a domain can be deleted.

## QA

### MAAS deployment
To run this branch you will need access to one of the following MAAS deployments:
- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch
You can run this branch by:
- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps
- Go to /MAAS/l/domain/2
- Try and delete to the domain and see there are no errors in the console

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/2132

## Backports
N/A
